.travis.yml

```diff
@@ -15,10 +15,12 @@ matrix:
   - go: tip
 
 before_install:
+- go get github.com/Masterminds/glide
 - go get github.com/mattn/goveralls
-- go get -u github.com/alecthomas/gometalinter
+- go get github.com/alecthomas/gometalinter
 
 install:
+- glide install
```

Comment:
I think it's a problem to have the vendor folder checked in AND do `glide install` here.
If you forget to check something into vendor the build won't fail even though it should.

Also I think it always makes sense to run `glide install` with `--strip-vendor`. To prevent issues like these: https://github.com/jtblin/kube2iam/pull/67